modules/client/src/addresslistVoting/internal/client/methods.ts

```diff
@@ -136,15 +136,14 @@ export class AddresslistVotingClientMethods extends ClientCore
     }
 
     const parsedLog = addresslistContractInterface.parseLog(log);
-    const proposalId: BigNumber = parsedLog.args["proposalId"];
+    const proposalId = parsedLog.args["proposalId"];
     if (!proposalId) {
       throw new ProposalCreationError();
     }
 
     yield {
       key: ProposalCreationSteps.DONE,
-      // TODO remove this when new proposal format
-      proposalId: proposalId.toNumber(),
+      proposalId: encodeProposalId(params.pluginAddress, Number(proposalId)),
```

Comment:
will `proposalId` be a base 10 number encoded in a string? I would make the numeric base explicit

Reply:
In other words, if `proposalId` is a number or a bigint, things will work fine. But if the value is `0x55ff` then, `Number()` will fail

Reply:
the value will be a number but ethers returns it as `BigNumberish`. Meaning that the ts typing system would complain if we just use it directly. Thus the usage of `Number(proposalId)`.
If you wan't I can add a `parseInt()` just to do it another way